plugins/net_plugin/net_plugin.cpp

```diff
@@ -98,7 +98,7 @@ namespace eosio {
       bool          have_block = false; // true if we have received the block, false if only received id notification
    };
 
-   struct by_block_id;
+   struct by_peer_block_id;
```

Comment:
This is fine, but why?

Reply:
compiler complaints about it being ambiguous after some includes were added

Reply:
Fair enough